org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/dom/parser/XMLScanner.java

```diff
@@ -366,24 +363,17 @@ TokenType internalScan() {
 				state = getWithinContentState();
 				return finishToken(offset, TokenType.PrologEnd);
 			}
-			
+
 			if (hasNextAttributeName()) {
 				state = ScannerState.AfterAttributeName;
 				return finishToken(offset, TokenType.AttributeName);
 			}
 
 			if (stream.advanceIfChar(_FSL)) { // /
-				state = ScannerState.WithinTag;
-				if(stream.advanceIfChar(_RAN)) { // >
-					state = ScannerState.WithinContent;
+				state = ScannerState.WithinContent;
+				if (stream.advanceIfChar(_RAN)) { // >
 					return finishToken(offset, TokenType.StartTagSelfClose);
 				}
-				return finishToken(offset, TokenType.Unknown);
-			}
-			int c = stream.peekChar();
-			if (c == _DQO || c == _SIQ) { // " || '
-				state = ScannerState.BeforeAttributeValue;
-				return internalScan();

```

Comment:
To prevent the issue I mentioned, I was thinking about adding something like this here:

```
int c = stream.peekChar();
if (c == _DQO || c == _SIQ) { // " || '
    int i = 1;
    int curr = stream.peekChar(i);
    while (curr != -1) {
        if (curr == c) {
            state = ScannerState.BeforeAttributeValue;
            return internalScan();
        }
        if (curr == _LAN) {
            break;
        }
        curr = stream.peekChar(i++);
    }
}
```

This is under the assumption that the `<` character cannot exist in an attribute value, and that `<` is always being interpreted as a start of a new element

Reply:
To be honnest with you, I tried to do the same code than HTML Language Service but there are some limitations too. I think we should write more tests with tolerant parser (liek your case) and we will see what we must to do. 

In other words, this PR should be done and improved after the release.